projects/layout/components/cell/cell.styles.less

```diff
@@ -81,6 +81,10 @@
     }
 
     &[data-size='s'] {
+        &:hover {
```

Comment:
I think we need to add borders to the cell, not just for hover state.

Reply:
fixed